infra/recipes/terraform/azure/modules/k8s/main.tf

```diff
@@ -1,16 +1,25 @@
 resource "azurerm_kubernetes_cluster" "aks_cluster" {
```

Comment:
Good to add description for each module to keep it consistent with other main.tf.

Reply:
Also a bit confused on meaning here. Can you clarify?

Reply:
Added example comment.